File: Testing/Code for Testing/Early Basic Syntax Testing/normalize/test_normalize.py
from dataclasses import dataclass, field
from math import sqrt

def _ul_sort(arr, n): arr[:n] = sorted(arr[:n])

class _ul_Exception(Exception):
    def __init__(self, type_name, message):
        self.type_name = type_name
        self.message = message
        super().__init__(message)


def main():
    counter = 10
    counter += 5
    counter *= 2
    i = 0
    i += 1
    i += 1
    i -= 1
    print(counter)
    print(i)
    val = 144.0
    deep = sqrt(sqrt(val))
    print(deep)
    x = 42
    y = float(float(x))
    print(y)
    a = 1
    b = 2
    c = 3
    if (((a > 0) and (b > 0)) and (c > 0)):
        print(a)
    return 0


if __name__ == "__main__":
    main()

File: Testing/Code for Testing/Early Basic Syntax Testing/normalize/test_normalize.js
const sqrt = Math.sqrt;


function main() {
    let counter = 10;
    counter += 5;
    counter *= 2;
    let i = 0;
    i++;
    i++;
    i--;
    console.log(counter);
    console.log(i);
    let val = 144.0;
    let deep = sqrt(sqrt(val));
    console.log(deep);
    let x = 42;
    let y = Number(Number(x));
    console.log(y);
    let a = 1;
    let b = 2;
    let c = 3;
    if ((((a > 0) && (b > 0)) && (c > 0))) {
        console.log(a);
    }
    return 0;
}

function __ul_sort(arr, n) { const sub = arr.slice(0, n).sort((a, b) => a < b ? -1 : a > b ? 1 : 0); for (let i = 0; i < n; i++) arr[i] = sub[i]; }

main();
